presto-native-execution/presto_cpp/main/operators/PartitionAndSerialize.cpp

```diff
@@ -142,10 +143,12 @@ class PartitionAndSerializeOperator : public Operator {
     }
   }
 
+  using TRowSize = uint32_t;
```

Comment:
nit: can we put this at the head of private section?

Reply:
sorry, I merged it internally already, but I'll submit a small follow up to move it.